AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/JoinCallActivity.java

```diff
@@ -61,69 +67,136 @@ protected void onDestroy() {
-                        final CharSequence charSequence, final int i, final int i1, final int i2) {
-                    // Do nothing
-                }
+        groupCallEditText = findViewById(R.id.group_call_text);
+        teamsMeetingEditText = findViewById(R.id.teams_meeting_text);
 
-                @Override
-                public void onTextChanged(final CharSequence charSequence, final int i, final int i1, final int i2) {
-                    // Do nothing
-                }
+        joinButton = findViewById(R.id.join_button);
 
-                @Override
-                public void afterTextChanged(final Editable editable) {
-                    if (editable.length() == 0) {
-                        //TODO: Error: "SPAN_EXCLUSIVE_EXCLUSIVE spans cannot have a zero length" occurs after
-                        // clearing text
-                        joinButton.setClickable(false);
-                        joinButton.setEnabled(false);
-                    } else {
-                        joinButton.setEnabled(true);
-                        joinButton.setClickable(true);
-                        if (!joinButton.hasOnClickListeners()) {
-                            joinButton.setOnClickListener(l -> joinCall());
-                        }
-                    }
-                }
-            });
-        editTextTextMeetingName.setOnFocusChangeListener((view, hasFocus) -> {
+        groupCallRadioButton = findViewById(R.id.group_call_radio_button);
+        teamsMeetingRadioButton = findViewById(R.id.teams_meeting_radio_button);
+
+        groupCallEditText.addTextChangedListener(onEditTextChanged());
+        groupCallEditText.setOnFocusChangeListener((view, hasFocus) -> {
             if (hasFocus) {
-                joinEnterText.setTextColor(ContextCompat.getColor(this, R.color.colorAccent));
+                groupCallRadioButton.setTextColor(ContextCompat.getColor(this, R.color.colorAccent));
             } else {
-                joinEnterText.setTextColor(ContextCompat.getColor(this, R.color.textbox_secondary));
+                groupCallRadioButton.setTextColor(ContextCompat.getColor(this, R.color.textbox_secondary));
+            }
+        });
+
+
+        teamsMeetingEditText.addTextChangedListener(onEditTextChanged());
+        teamsMeetingEditText.setOnFocusChangeListener((view, hasFocus) -> {
+            if (hasFocus) {
+                teamsMeetingRadioButton.setTextColor(ContextCompat.getColor(this, R.color.colorAccent));
+            } else {
+                teamsMeetingRadioButton.setTextColor(ContextCompat.getColor(this, R.color.textbox_secondary));
+            }
+        });
+
+        groupCallRadioButton.setOnClickListener(l -> {
+            groupCallEditText.setVisibility(View.VISIBLE);
+            groupCallEditText.requestFocus();
+            if (teamsMeetingRadioButton.isChecked()) {
+                teamsMeetingRadioButton.setChecked(false);
+                teamsMeetingEditText.getText().clear();
```

Comment:
since teamsMeetingEditText and groupCallEditText are two different EditText, can we keep the existing values when toggling between Group and Teams?